code/hsec-tools/src/Security/Advisories/Git.hs

```diff
@@ -117,4 +119,7 @@ getAdvisoryGitInfo path = do
       -- the same as `ExitFailure`
       pure . Left $ GitProcessError status stdout stderr
   where
-    parseTime s = maybe (Left $ GitTimeParseError s) Right $ iso8601ParseM s
+    parseTime :: String -> Either GitError ZonedTime
+    parseTime s = maybe (Left $ GitTimeParseError s) Right $
+       iso8601ParseM s
```

Comment:
We should force the format at git level: https://stackoverflow.com/a/7651782

Reply:
I tried to pass `--date=local` and it didn't work. This is in general not really ideal, we shouldn't depend on the git installation if git itself doesn't guarantee a stable interface to these things and/ or this is not documented. 

There's also https://github.com/haskell/time/issues/257 which I don't know whether it's the case.

Reply:
https://github.com/MangoIV/cabal-audit/actions/runs/9321254777/job/25659857545

here's a run. the build step is `Test git` which uses the date format local and it still outputs a non-zoned iso8601. I wouldn't have done the separate parse try without trying that first.

Reply:
*I assume* (read "guess") that it *is* valid to output a non-zoned iso8601 even with `local` *if* the computer is itself in `UTC` which is why I'm not sure whether parsing an unzoned time into a `UTCTime` should just work.

Reply:
Let's do this, it seems to be the lesser bad solution we have.

Reply:
I'm confused, setting it in git is a non-solution because it doesn't work. We *have* to do it like this, no?

Reply:
I'm confused, I thought that, requiring a ZonedTime from git wasn't working, requiring a UTCTime doesn't work either?

Reply:
@frasertweedale was talking of a flag few minutes ago (at ZuriHac), maybe we can have a try